Missing dependencies when using the globally install helix-cli

**Description**
`@babel/*` dependencies are missing with using the globally installed helix-cli.

**To Reproduce**
I use project https://github.com/adobe/helix-pages:
1. `npm install -g @adobe/helix-cli`
2. In the project, run `hlx clean; hlx build; hlx package`
3. Then, run `ll .hlx/build/html.zip`: ` 8920814 Jun  7 08:29 .hlx/build/html.zip`
4. Deploy and publish the project, I get activation error:

```
"error": "Initialization has failed due to: Error: Cannot find module '@babel/runtime-corejs2/helpers/interopRequireDefault'\n    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)\n    at Function.Module._load (internal/modules/cjs/loader.js:507:25)\n    at Module.require (internal/modules/cjs/loader.js:637:17)\n    at require (internal/modules/cjs/helpers.js:22:18)\n    at Object.<anonymous> (/nodejsAction/XXbedOKc/node_modules/@adobe/htlengine/node_modules/xregexp/lib/index.js:3:30)\n    at Module._compile (internal/modules/cjs/loader.js:689:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)\n    at Module.load (internal/modules/cjs/loader.js:599:32)\n    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)\n    at Function.Module._load (internal/modules/cjs/loader.js:530:3)"
```

Now do:

1. `git clone https://github.com/adobe/helix-cli.git`
2. `cd helix-cli`
3. `npm install; npm link`
4. In the project, run `hlx clean; hlx build; hlx package`
3. Then, run `ll .hlx/build/html.zip`: `10193801 Jun  7 08:14 .hlx/build/html.zip`
4. Deploy and publish the project, works as expected!

At least `@babel/*` dependencies are missing in the first case. 12Mo of dependencies seem to be missing.

**Version:**
run: `$ hlx --version`
4.2.2 in both cases

cc @tripodsan

--- a/package.json
+++ b/package.json
@@ -55,6 +55,7 @@
     "hyperscript": "^2.0.2",
     "ignore": "^5.1.0",
     "ini": "^1.3.5",
+    "ip-regex": "^4.1.0",
     "junit-report-builder": "1.3.1",
     "lodash": "4.17.11",
     "mime-types": "2.1.24",
@@ -77,6 +78,7 @@
     "uri-js": "4.2.2",
     "uuid": "3.3.2",
     "webpack": "4.33.0",
+    "xregexp": "^4.2.4",
     "yargs": "13.2.2"
   },
   "devDependencies": {
@@ -110,8 +112,7 @@
     "proxyquire": "^2.1.0",
     "semantic-release": "^15.13.12",
     "sinon": "7.3.2",
-    "unzip2": "0.2.5",
-    "xregexp": "^4.2.4"
+    "unzip2": "0.2.5"
   },
   "snyk": true,
   "lint-staged": {

--- a/src/parcel/packager-config.js
+++ b/src/parcel/packager-config.js
@@ -17,7 +17,7 @@ module.exports = {
     openwhisk: '3.18.0',
     'body-parser': '1.18.3',
     'cls-hooked': '4.2.2',
-    request: '2.88.0',
+    // request: '2.88.0',
     'request-promise': '4.2.2',
 
     // webpack isn't really provided by the container, but it injects itself into the list of

